### DISCUSSION TITLE
How about we let the `pass` dispatcher pass another key combination

### DISCUSSION THREAD

**Discussion by tmarkov** [hyprwm/Hyprland#3510] [General]
Currently, the pass dispatcher always passes the keys it's bound to. That works great for the case when your goal is to make a global keybind, however, it limits some other potential use cases.

I think it would be good if we could set what exactly gets passed. For example, `bind = $mainMod, s, pass, MyApp, Ctrl, f` (or some other syntax). This would also allow using the `pass` dispatcher with `hyprctl` (which currently seems to execute, but not do much).

Some use cases could be:
1. Some apps don't have a control utility, but can be controlled by keyboard shortcuts. This would allow controlling them though `hyprland` configuration.
2. Some apps don't allow changing the default keybinds. We could use this to remap them.

**Comment by v4u6h4n** [comment]
So does `hyprctl dispatch pass` do anything? I've tried a lot of permutations to make sure my syntax is right and nothing seems to work.

**Comment by joshheyse** [comment]
This would be an amazing feature for my use case. I am constantly switching between Hyprland on  fedora and macOS. The ability to standardize keyboard shortcuts at a global level across the two systems would save me so many cycles.

**Comment by joshheyse** [comment]
I've added a feature request with a proposed implementation at https://github.com/hyprwm/Hyprland/issues/5631
